test/sai_test/config/lag_configer.py

```diff
@@ -0,0 +1,194 @@
+    """
+
+    def __init__(self, test_obj) -> None:
+        """
+        Init Lag configrer.
+        
+        Args:
+            test_obj: the test object
+        """
+        self.test_obj = test_obj
+        self.client = test_obj.client
+    
+    def create_lag(self, lag_port_idxs):
+        """
+        Create lag and its members.
+
+        Args:
+            lag_port_idxs: lag port indexs
+
+        Returns:
+            Lag: lag object
+        """
+
+        lag = Lag()
+        lag_id = sai_thrift_create_lag(self.client)
+        lag_members = self.create_lag_member(lag_id, lag_port_idxs)
+        self.test_obj.assertEqual(self.test_obj.status(), SAI_STATUS_SUCCESS)
+        lag.lag_id = lag_id
+        lag.lag_members = lag_members
+        return lag
+    
+    def create_lag_member(self, lag_id, lag_port_idxs):
+        """
+        Create lag members for a lag.
+
+        Args:
+            lag: lag object
+            lag_port_idxs: lag member port indexs
+
+        Returns:
+            lag_members: list of lag_member
+        """
+
+        lag_members = []
+        for port_index in lag_port_idxs:
+            lag_member = sai_thrift_create_lag_member(self.client, 
+                                                      lag_id=lag_id, 
+                                                      port_id=self.test_obj.port_list[port_index])
+            self.test_obj.assertEqual(self.test_obj.status(), SAI_STATUS_SUCCESS)
+            lag_members.append(lag_member)
+        return lag_members
+    
+    def set_lag_hash_algorithm(self, algo=SAI_HASH_ALGORITHM_CRC):
+        """
+        Set lag hash algorithm.
+
+        Args:
+            algo (int): hash algorithm id
+        """
+        sai_thrift_set_switch_attribute(self.client, lag_default_hash_algorithm=algo)
+
+    def set_lag_hash_attribute(self):
+        """
+        Set lag hash attributes.
+
+        Args:
+            hash_fields_list: hash fields list
+        """
+        attr_list = sai_thrift_get_switch_attribute(self.client, lag_hash=True)
+        lag_hash_id = attr_list['SAI_SWITCH_ATTR_LAG_HASH']
+
+        hash_fields_list = [SAI_NATIVE_HASH_FIELD_SRC_IP,
+                            SAI_NATIVE_HASH_FIELD_DST_IP,
+                            SAI_NATIVE_HASH_FIELD_IP_PROTOCOL,
+                            SAI_NATIVE_HASH_FIELD_L4_DST_PORT,
+                            SAI_NATIVE_HASH_FIELD_L4_SRC_PORT]
+
+        hash_attr_list = sai_thrift_s32_list_t(count=len(hash_fields_list), int32list=hash_fields_list)
+        sai_thrift_set_hash_attribute(self.client, lag_hash_id, native_hash_field_list=hash_attr_list)
+
+    def set_lag_hash_seed(self, seed=400):
+        """
+        Set lag hash seed.
+
+        Args:
+            seed: hash seed value
+        """
+        sai_thrift_set_switch_attribute(self.client, lag_default_hash_seed=seed)
+
+    def create_route_and_neighbor_entry_for_lag(self, lag_id, ip_addr, mac_addr, port_id):
```

Comment:
please move this method to a route configer, and named as create_route_and_neighbor_entry_for_port

Reply:
if we transfer the port object id, then we can use this method for both normal ports and lag ports?